test/regress/regress0/quantifiers/issue2033-macro-arith.smt2

```diff
@@ -1,4 +1,4 @@
-; COMMAND-LINE: --macros-quant
```

Comment:
how come this is needed now and was not needed when it was a preprocessing pass? This is a bit of a regression, right? I am guessing that is used to be able to check models and now it cannot?

Reply:
Yes, this has revealed a more general issue with check-models, that we don't evaluate terms under quantifiers.  This will be addressed on future PRs.